pixi.toml

```diff
@@ -0,0 +1,123 @@
+store-golden-master = "python tests/glm/test_golden_master.py"
+store-benchmark-golden-master = "python tests/glm/test_benchmark_golden_master.py"
+
+[feature.docs.tasks]
+postinstall-docs = "pip install --no-use-pep517 --no-build-isolation --no-deps --disable-pip-version-check -e ."
+make-docs = "cd docs && make html"
+serve-docs = { cmd = "python -m http.server --directory docs/_build/html", depends_on = ["make-docs"] }
+
+[feature.tutorial.tasks]
+postinstall-tutorial = "pip install --no-use-pep517 --no-build-isolation --no-deps --disable-pip-version-check -e ."
+
+[feature.benchmark.tasks]
+postinstall-benchmark = "pip install --no-use-pep517 --no-build-isolation --no-deps --disable-pip-version-check -e ."
+glm-benchmarks-run = "glm_benchmarks_run"
+glm-benchmarks-analyze = "glm_benchmarks_analyze"
+
+[build-dependencies]
+c-compiler = "*"
+cxx-compiler = "*"
+cython = "*"
+setuptools-scm = ">=8.1"
+
+[host-dependencies]
+python = ">=3.9"
+pip = "*"
+
+[dependencies]
+numexpr = "*"
+pandas = ">=1.1"
+tabmat = ">=4.0.0"
+scikit-learn = ">=0.23"
+scipy = "*"
+tqdm = "*"
+formulaic = "*"
+pyarrow = "*"
+
+[feature.test.dependencies]
+statsmodels = "*"
+pytest = "*"
+pytest-xdist = "*"
+git_root = "*"
+click = "*"
+attrs = "*"
+psutil = "*"
+
+[feature.dev.dependencies]
+ruff = "*"
+ipdb = "*"
+ipython = "*"
+line_profiler = "*"
+memory_profiler = "*"
+pre-commit = "*"
+
+[feature.docs.dependencies]
+jinja2 = "*"
+jupyterlab = "*"
+jupytext = "*"
+make = "*"
+matplotlib-base = "*"
+nbclassic = ">=0.2.8"
+nbsphinx = ">=0.8.3"
+sphinx = ">=3.5.3"
+sphinx_rtd_theme = "*"
+sphinxcontrib-apidoc = "*"
+sphinxext-altair = "*"
+
+[feature.tutorial.dependencies]
+jupyterlab = "*"
+altair = "*" # used in docs/tutorials/rossman
+openml = "*" # used for downloading datasets in the tutorials.
+dask-ml = ">=2022.5.27" # used in tutorials rossman and insurance
+shapely = "*" # used in docs/tutorials/regularization_housing_data
+libpysal = "*" # used in docs/tutorials/regularization_housing_data
+
+[feature.benchmark.dependencies]
+h2o-py = "*"
+openjdk = "*"
+
+[feature.py39.dependencies]
+python = "3.9.*"
+[feature.py310.dependencies]
+python = "3.10.*"
+[feature.py311.dependencies]
+python = "3.11.*"
+[feature.py312.dependencies]
+python = "3.12.*"
+
+[feature.oldies]
+platforms = ["linux-64"]
+[feature.oldies.dependencies]
+python = "3.9.*"
+pandas = "1.1.*"
+scikit-learn = "0.23.*"
+tabmat = "4.0.*"
+
+[environments]
+default = ["dev", "test"]
+docs = ["docs", "tutorial"]
+benchmark = ["benchmark"]
+py39 = ["py39", "test"]
+py310 = ["py310", "test"]
+py311 = ["py311", "test"]
+py312 = ["py312", "test"]
+oldies = ["oldies", "test"]
+
+[tool.update]
+autoupdate-branch = "pixi-update"
+autoupdate-commit-message = "Update pixi lockfile"
+autoupdate-pull-request-labels = ["dependencies"]
+autoupdate-schedule = "monthly" # weekly, monthly, quarterly, never
+ignore-environments = []
+ignore-platforms = []
```

Comment:
These are the default values. If you don't plan on specifying them, I would just suggest leaving them out. The updater will come even if you don't specify `[tool.update]`.